KEP 1645: slightly relax the managed by requirements on EndpointSlices (#4930)
kubernetes/enhancements: keps/sig-multicluster/1645-multi-cluster-services-api/README.md

the underlying `Service` to be created in each importing cluster within the
clusterset, associated with the derived `ServiceImport`. One or more
`EndpointSlice` resources will exist for the exported `Service`, with each
`EndpointSlice` containing only endpoints from a single source cluster. An

> `EndpointSlice` created by a mcs-controller must not be marked as managed by the
> default `EndpointSlice` controller to avoid any conflicts between both controllers.

Reviewer: ```suggestion
`EndpointSlice` created by an mcs-controller must not be marked as managed by the
default `EndpointSlice` controller to avoid any conflicts between the controllers.
```

> `EndpointSlice` created by an mcs-controller must not be marked as managed by the
> default `EndpointSlice` controller to avoid any conflicts between the controllers.

Reviewer 2: This allows `EndpointSlice` objects with no `managed-by` label at all… How about

```suggestion
`EndpointSlice` created by an mcs-controller must be marked as managed by the
mcs-controller, not the default `EndpointSlice` controller to avoid any conflicts
between the controllers.
```

Author: Indeed that makes sense, thanks!
